tests/test_models.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Tests the models module."""
```

Comment:
Maybe also add some tests for the various PydanticCustomErrors.

Reply:
I think I have tested all of them in the test_load module. Are you missing any?

Reply:
2 or 3 are missing, see [here](https://coveralls.io/builds/64059853/source?filename=src%2Fschemapack%2Fmodels.py)

Reply:
We can also add it later when this is really used and not just a PoC.